cryostat/values.yaml

```diff
@@ -121,7 +121,7 @@ datasource:
     ## @param datasource.image.pullPolicy Image pull policy for the JFR Data Source container image
     pullPolicy: Always
     ## @param datasource.image.tag Tag for the JFR Data Source container image
-    tag: "latest"
+    tag: "2.2"
```

Comment:
Same question as above with Cryostat.

Reply:
I explained some context for these choices here:

https://github.com/cryostatio/cryostat-operator/pull/480

whichever way we decide, I will update the relevant PR(s).